types/gapi.client.datacatalog-v1/package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "private": true,
+  "dependencies": {
+    "@maxim_mazurok/gapi.client.datacatalog-v1": "latest"
```

Comment:
I know DT maintainers area againsts blanket dependencies. Can this be at least "1"? (or anything you want to lock to v1)

Reply:
**TL;DR** version part is a plain string from google and can't be parsed into a number.

More context from https://github.com/microsoft/DefinitelyTyped-tools/issues/505:
> I'll be changing the naming pattern of `@types/gapi.client.*` packages in order to accommodate multiple versions that can't be described as `${major}.${minor}`, such as `v2beta1`, `directory_v1`, `reports_v1`, etc.

You can also see that the original issue that prompted this change was because Google uses `reports_v1` and `directory_v1` as "versions" of the `admin` API (see https://github.com/Maxim-Mazurok/google-api-typings-generator/issues/652). I believe it's a misuse from their side, but we have to support this.

You can also find some other weird version patterns in this integration test: https://github.com/Maxim-Mazurok/google-api-typings-generator/blob/261346a336932191aab8b838fcee21ceae7d29da/test/discovery.integration.spec.ts#L108-L288

Not sure what you mean by "blanket dependencies"? It was previously agreed with DT maintainers to have a reference to `@maxim_mazurok/gapi.client.*` in order to avoid me spamming DT with PRs as APIs update hourly.

Reply:
I mean, we have issues with types, that get outdated with time, while still using 'latest' in `package.json`, hence loading newest version of packages from dependency graph.
Also,  just as security measure.

Reply:
Mm, I think that when you do `npm update` - it should update the whole dependency tree, no?

Anyway, this is the way it is set up now. This way was suggested by [Eli Barzilay](https://github.com/elibarzilay):
> push your own typing package to npm and make the DT one depend on it
https://github.com/DefinitelyTyped/dt-mergebot/issues/203#issuecomment-715645986

However if this is a blocker - I can make it `0.0.*` instead of `latest` because in the new packages major and minor version is always zero (see https://github.com/Maxim-Mazurok/google-api-typings-generator/blob/da13487140204a3fa3bee0ca861f6cb480799bd5/src/utils.ts#L35) and only patch version is updated to represent the revision number we get from google, `20220831` for example.

Reply:
I can see it's already there in existing code for `gapi`. Except of `gapi` it's only allowed in 3 different types (probably as omission, not a rule). Let's skip this discussion